### DISCUSSION TITLE
Options Screen is empty

### DISCUSSION THREAD

**Discussion by jainv4156** [oppia/oppia-android#5567] [General Discussion]
I installed my Oppia Android app following the documentation, and the app is installed successfully. However, the options menu screen is empty in both the administrator and custom profiles.

The app is running on a Pixel 5 with API 30 in debug build

**Comment by adhiamboperes** [comment]
@jainv4156, this is a known issue. The app use a custom language implementation, and the language module is only available in Bazel. And since the options menu attempts to fetch the app and audio languages, it fails in Gradle builds, leading to the blank screen.

**Reply by jainv4156** [reply]
Thank you for the clarification!
